src/web/App.tsx

```diff
@@ -24,12 +24,11 @@ function AppContent() {
   const { LoggedInUser } = useContext(CurrentUserContext);
   const { participant } = useContext(ParticipantContext);
   const isLocalDev = process.env.NODE_ENV === 'development';
-  const location = useLocation();
 
   if (LoggedInUser?.user?.participants!.length === 0) {
     return <ErrorView message='You do not have access to any participants.' />;
   }
-  if (location.pathname !== '/account/create' && LoggedInUser && !participant) {
+  if (LoggedInUser && !participant) {
```

Comment:
this account/create page does not exist anymore and should be removed. randomly found this when testing stuff